repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -3297,7 +3305,7 @@ public void updateClassifications(EntityMutationContext context, String guid, Li
             String      entityGuid = graphHelper.getGuid(vertex);
             AtlasEntity entity     = instanceConverter.getAndCacheEntity(entityGuid, ENTITY_CHANGE_NOTIFY_IGNORE_RELATIONSHIP_ATTRIBUTES);
 
-            if (isActive(entity)) {
```

Comment:
Why this change?

Reply:
Cause if we are adding the classification we will always keep it in the classiifcation text?

Reply:
Skipping archived entities again makes __classificationText inconsistent

Reply:
So we want to discover archived entity if we have tag attached? right